src/Opdex.Platform.WebApi/Models/Requests/Vaults/VaultCertificateFilterParameters.cs

```diff
@@ -8,10 +8,10 @@ namespace Opdex.Platform.WebApi.Models.Requests.Vaults;
 public sealed class VaultCertificateFilterParameters : FilterParameters<VaultCertificatesCursor>
 {
     /// <summary>
-    /// Address of the certificate holder.
+    /// Address of the certificate owner.
     /// </summary>
     /// <example>tTTuKbCR2UnsEByXBp1ynBz91J2yz63h1c</example>
-    public Address Holder { get; set; }
```

Comment:
Renamed `Holder` to `Owner` in the _query_ parameters of `GET /vaults/{vault}/certificates`